src/js/utils/guid.js

```diff
@@ -1,17 +1,19 @@
 /**
  * @file guid.js
- *
+ * @module guid
+ */
+
+/**
  * Unique ID for an element or function
  * @type {Number}
- * @private
  */
 let _guid = 1;
 
 /**
- * Get the next unique ID
+ * Get the next unique ID by incrementing the global _guid singleton.
```

Comment:
This addition seems unclear to me. It's not really a singleton, nor is it global. Perhaps just "Get a unique, auto-incrementing ID number that has not been returned before." or something along those lines?